components/inlinealert/stories/inlinealert.stories.js

```diff
@@ -0,0 +1,57 @@
+// Import the component markup template
+import { Template } from "./template";
+
+export default {
+  title: "In-line Alert",
+  description: "In-line alerts display a non-modal message associated with objects in a view. These are often used in form validation, providing a place to aggregate feedback related to multiple fields.",
+  component: "InLineAlert",
+  argTypes: {
+    headerText: {
+      name: "Header Text",
+      type: { name: "string", required: true },
+      table: {
+        type: { summary: "string" },
+        disable: false,
+        category: "Content",
+      },
+      control: { type: "text" },
+    },
+    text: {
+      name: "Text",
+      type: { name: "string", required: true },
+      table: {
+        type: { summary: "string" },
+        disable: false,
+        category: "Component",
+      },
+      control: { type: "text" },
+    },
+    variant: {
+      name: "Variants",
+      type: { name: "string" },
+      table: {
+        type: { summary: "string" },
+        category: "Component",
+      },
+      options: ["neutral", "info", "positive", "notice", "negative", "closable"],
```

Comment:
Is closable in the same category as the other variants? I'm wondering if you can have a closable alert that is also neutral.

Reply:
Oh, great catch. In our docs site we have "An in-line alert with a close button in the footer. Combine this strategy with any variant" as the definition of Closable, and the actual Spectrum site doesn't have Closable at all...

I'm going to switch it from a Variant to some other kind of option!